client/my-sites/checkout/checkout/redirect-payment-box.jsx

```diff
@@ -232,6 +232,6 @@ class SourcePaymentBox extends PureComponent {
 		return paymentMethodName( this.props.paymentType );
 	}
 }
-SourcePaymentBox.displayName = 'SourcePaymentBox';
+RedirectPaymentBox.displayName = 'RedirectPaymentBox';
```

Comment:
Since `RedirectPaymentBox` is a class, we can now define this as a static member variable inside of the class:
```
static displayName = 'RedirectPaymentBox';
```